src/components/enterprise-user-subsidy/data/hooks/hooks.js

```diff
@@ -244,3 +245,21 @@ export function useCustomerAgreementData(enterpriseId) {
 
   return [customerAgreement, isLoading];
 }
+
+const getRedeemablePoliciesData = async ({ queryKey }) => {
+  const enterpriseId = queryKey[1];
+  const userID = queryKey[2];
+  const response = await fetchRedeemableLearnerCreditPolicies(enterpriseId, userID);
+  return camelCaseObject(response.data);
+};
+
+export function useRedeemableLearnerCreditPolicies(enterpriseId, userID) {
+  const { data: redeemableLearnerCreditPolicies, isLoading, error } = useQuery({
```

Comment:
[suggestion] Note: It's usually a [good idea](https://tkdodo.eu/blog/react-query-data-transformations#2-in-the-render-function) for the hook to return everything from `useQuery`, so that it doesn't require any additional maintenance to expose specific fields from `useQuery` as they become necessary in the UI.

Example:

https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/6fc25f0bfbd988b6805d2bee274a46d39f15eef3/src/components/course/data/hooks.jsx#L574-L606

https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/6fc25f0bfbd988b6805d2bee274a46d39f15eef3/src/components/course/CoursePage.jsx#L104-L121